### ISSUE TITLE
Positive Integer values always packed as 8 octal length (64-bit) values. Breaks 32-bit tools like snmpget and snmpwalk on Windows.

### ISSUE THREAD

**Issue by omkger** [delimitry/snmp-server#33]
_write_int() is generating 8 octet length integer tlv for all integers (greater than zero), even if it takes less than 8 octets to represent them.

I realize there is handling of negative integers going on.
Partial fix is to always enable strip_trailing_zeroes. A complete fix would replicate the functionality that exists for negative numbers, for positive numbers. i.e. pack the appropriate size of bytes for the integer.

Partial fix:
>add the line "strip_leading_zeros = True" at the beginning of _write_int() function

Also, there is a slight discrepancy between integer() and _write_int()

After this fix, integers on win32 netsnmp tools should work until you assign an integer value larger than 32 bits in your MIB (server config), just a heads up for everyone who is using precompiled win32 netsnmp tools on Windows.

tldr;
64-bit (8 octet) python integers making 32-bit compiled win32 netsnmp tools shart themselves.

**Comment by oschwartz10612** [comment]
I think that #40 resolves this issue - at least in my testing. If you have a chance can you take a look as well?
